EIPS/eip-4973.md

```diff
@@ -185,7 +185,25 @@ You can find an implementation of this standard in [../assets/eip-4973](../asset
 
 ## Security Considerations
 
-There are no security considerations related directly to the implementation of this standard.
+### Privacy Considerations
+
+#### Storing Personally Identifiable Information (PII) on-chain presents privacy risks
+
+Data stored within an account-bound token often times are claims which are directly associated with an account. In general, it's a privacy concern to directly store PII related to a person on-chain or to link data directly to an identifier like an Ethereum account. Doing so would make it impossible for a user to invoke their right to be forgotten which is necessary under GDPR and could place dApp developers under scrutiny if they do so. Even in the case where data is generally regarded as public knowledge, public attributes contained in an account-bound token shouldn't be published on chain as this can lead unexpected behavior that could leak user behavior or insights which can be used to later harm the controller of the account. Instead, dApp developers should consider providing an opaque URL to an off chain storage solution so that the data is not published immutably on chain so that the claim can either be updated, refuted, or forgotten in compliance with GDPR. 
```

Comment:
This section should be entirely removed:

- We're assuming GDPR and "right to be forgotten" regime (EU), but EIP readers/users are international. Generally, it is not our place to tell people what to do.

It is a good idea to suggest putting a claim off-chain to a mutable data source:

> Instead, dApp developers should consider providing an opaque URL to an off chain storage solution so that the data is not published immutably on chain so that the claim can either be updated, refuted, or forgotten in compliance with GDPR. 

We should leave this in but rephrase it away from GDPR-motivated framing.